### PR TITLE
Helm: weekly r196

### PR DESCRIPTION
**Pull request by krajorama**
Signed-off-by: György Krajcsovits <gyorgy.krajcsovits@grafana.com>

### CODE REVIEW

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM. In some previous weeklies of the helm chart we've also updated the images of mimir to use the weekly releases. Do we want to do that going forward or not?

**Comment by krajorama** [comment]
> LGTM. In some previous weeklies of the helm chart we've also updated the images of mimir to use the weekly releases. Do we want to do that going forward or not?

Good question. Would be nice to tweak just one parameter, but people probably expect new software as well.

**Comment by krajorama** [comment]
I'll update the images as well

**Comment by krajorama** [comment]
On hold until we discuss the alertmanager external_url change

File: operations/helm/charts/mimir-distributed/templates/validate.yaml [hunk 1]
```diff
@@ -130,3 +130,8 @@
     {{- fail "memcached-results.enabled has been renamed to results-cache.enabled" }}
   {{- end }}
 {{- end }}
+
+
+{{- if eq ($calculatedConfig.alertmanager).external_url "/alertmanager" }}
+  {{- fail "Do not set alertmanager.external_url to the old invalid value /alertmanager in either mimir.config or mimir.structuredConfig. The value must start with proper http:// or https:// scheme." }}
+{{- end }}
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/templates/validate.yaml:137]: I'm not too sure about validating config here that mimir already validates. Aren't we just duplicating logic like that?

**Comment by krajorama** [comment]
We've agree to improve the validation logic so external_url setting can remain the same and not break. https://github.com/grafana/mimir/pull/2542

**Review by dimitarvdimitrov — APPROVED** [review]

**Pull request merged**